tests/app/celery/test_scheduled_tasks.py

```diff
@@ -1054,7 +1054,13 @@ def test_letter_not_raise_alert_if_ack_files_match_zip_list(mocker, notify_db):
 
     letter_raise_alert_if_no_ack_file_for_zip()
 
+    yesterday = datetime.utcnow() - timedelta(days=1)
+    subfoldername = datetime.utcnow().strftime('%Y-%m-%d')
     assert mock_file_list.call_count == 2
+    assert mock_file_list.call_args_list == [
+        call(bucket_name='test-letters-pdf', subfolder=subfoldername, suffix='.zip'),
+        call(bucket_name='test.notify.com-ftp', subfolder='root/dispatch', suffix='.ACK.txt', last_modified=yesterday),
```

Comment:
Could use the actual config name to be clearer - `current_app.config['LETTERS_PDF_BUCKET_NAME']` and `current_app.config['DVLA_RESPONSE_BUCKET_NAME']`

Reply:
Changed